### PR TITLE
fix(amplify-console-hosting): printout more informative message

### LINKED ISSUE #4086: What is the default hosting for an app?
>**Note**: If your question is regarding the AWS Amplify Console service, please log it in the 
[official AWS Amplify Console forum](https://forums.aws.amazon.com/forum.jspa?forumID=314&start=0)

**Which Category is your question related to?**

**Amplify CLI Version** 

You can use `amplify -v` to check the amplify cli version on your system

4.18.0

**What AWS Services are you utilizing?**

**Provide additional details e.g. code snippets**
When creating a new amplify app, prior to ever running `amplify add hosting`, how is my app hosted? In Route53, I see DNS records pointing to CloudFront, but I don't see a distribution in the CloudFront console.

### PR DESCRIPTION
**Pull request by UnleashedMind**
When frontend is created in Amplify Console, but hosting is not added locally, if the user execute
amplify hosting remove, this update will print out a more informative error message

fix #4086

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/amplify-console-hosting/hosting/index.js [hunk 1]
```diff
@@ -13,8 +13,9 @@ const HELP_INFO_PLACE_HOLDER =
   'Manual deployment allows you to publish your web app to the Amplify Console without connecting a Git provider. Continuous deployment allows you to publish changes on every code commit by connecting your GitHub, Bitbucket, GitLab, or AWS CodeCommit repositories.';
 const REMOVE_ERROR_MESSAGE = 'There was an error removing the auth resource';
 const HOSTING_NOT_ENABLED = 'Amplify Console hosting is not enabled.';
-const HOSTING_ALREADY_ENABLED =
-  'Amplify Console hosting has already been enabled';
+const HOSTING_ONLY_ENABLED_ONLINE =
```
Comment (jhockett) [inline packages/amplify-console-hosting/hosting/index.js:16]: Nit: The name of this `const` could be more clear. What about `HOSTING_ENABLED_IN_CONSOLE`?

File: packages/amplify-console-integration-tests/src/consoleHosting/constants.ts [hunk 2]
```diff
@@ -9,3 +9,6 @@ export const TYPE_UNKNOWN = 'unknown';
 export const ORIGINAL_ENV = 'integtest';
 export const NEW_ENV = 'test';
 export const PROVIDER = 'awscloudformation';
+export const HOSTING_ONLY_ENABLED_ONLINE =
```
Comment (jhockett) [inline packages/amplify-console-integration-tests/src/consoleHosting/constants.ts:12]: Same nit as above.

File: packages/amplify-console-hosting/hosting/index.js (hunk 1 above)
Comment (UnleashedMind) [reply packages/amplify-console-hosting/hosting/index.js]: looks good, changed.

File: packages/amplify-console-integration-tests/src/consoleHosting/constants.ts (hunk 2 above)
Comment (UnleashedMind) [reply packages/amplify-console-integration-tests/src/consoleHosting/constants.ts]: looks good, changed.

**Comment by brene** [comment]
Message looks good to me.

**Pull request merged**